doc/run_experiment.rst

```diff
@@ -506,8 +506,18 @@ fixed_parameters *(Optional)*
 """""""""""""""""""""""""""""
 
 List of dicts containing parameters you want to have fixed for each
-classifier in :ref:`learners` list. Any empty ones will be ignored
-(and the defaults will be used).
+learner in :ref:`learners` list. Any empty ones will be ignored
+(and the defaults will be used). If there is a conflict between parameters
+specified in :ref:`param_grids` and :ref:`fixed_parameters`, an exception
+will be raised (unless the conflicting values are the same). Furthermore,
```

Comment:
I agree, maybe "conflict" is the wrong word. I just meant the situation where a parameter is both referred to in the fixed parameters and in the parameter grid. This is a bit of an unusual situation, probably. And the only way they can agree is by specifying a list containing a single value in the parameter grid. To me, that seems like the same thing as just fixing its value. Anyway, maybe I can just rephrase it?